packages/bridge-ui/src/store/transactions.ts

```diff
@@ -6,12 +6,5 @@ import type { BridgeTransaction, Transactioner } from '../domain/transactions';
 const pendingTransactions = writable<Transaction[]>([]);
 const transactions = writable<BridgeTransaction[]>([]);
 const transactioner = writable<Transactioner>();
-const showTransactionDetails = writable<BridgeTransaction>();
-const showMessageStatusTooltip = writable<boolean>();
```

Comment:
`showTransactionDetails` and `showMessageStatusTooltip` can perfectly be local variables within `Transactions` component